### PR TITLE
[MURG-256] API documentation reference to smartsheet domain: TLD

### PR DESCRIPTION
**Pull request by aiyer-smartsheet**
As part of the TLD (top level domain) changes and migration, API calls/integration should be done against api.rm.smartsheet.com
Updated the documentation to reflect the same.

Note: This does not include changes to vnext/beta.

### CODE REVIEW

**Comment by tithonium** [comment]
The repo structure has changed and this PR will need to be updated. Let me know if you need assistance.

**Comment by tithonium** [comment]
@aiyer-smartsheet I rebased and resolved conflicts, and pushed to https://github.com/10Kft/10kft-api/tree/feature/api-domain-change-rebased. If you want to accept the changes as is, you can just `git checkout feature/api-domain-change ; git reset --hard origin/feature/api-domain-change-rebased ; git push -f` or something similar. Either way, feel free to delete the `feature/api-domain-change-rebased` branch afterward.

**Comment by aiyer-smartsheet** [comment]
> @aiyer-smartsheet I rebased and resolved conflicts, and pushed to https://github.com/10Kft/10kft-api/tree/feature/api-domain-change-rebased. If you want to accept the changes as is, you can just `git checkout feature/api-domain-change ; git reset --hard origin/feature/api-domain-change-rebased ; git push -f` or something similar. Either way, feel free to delete the `feature/api-domain-change-rebased` branch afterward.

Thank you! :)

**Comment by baonguyen84** [comment]
Make sense to me. Question though, I do not know who own this repo and how it will be deploy to be public, so ... after we merge I assume it will be public, right? What is the plan for this? I mean, will we just merge on GA day or what? Thank @aiyer-smartsheet

**Comment by aiyer-smartsheet** [comment]
> Make sense to me. Question though, I do not know who own this repo and how it will be deploy to be public, so ... after we merge I assume it will be public, right? What is the plan for this? I mean, will we just merge on GA day or what? Thank @aiyer-smartsheet

That has been my understanding too, merging this should make it public.
Plan is to deploy this on 8/11 (TLD GA date)

**Comment by tithonium** [comment]
@aiyer-smartsheet @baonguyen84 
> That has been my understanding too, merging this should make it public.

It's been over a month since I touched it, but I believe you now need to run `rake publish` to make it 'public' after committing. See the README and #74 for more information.

**Comment by aiyer-smartsheet** [comment]
> @aiyer-smartsheet @baonguyen84
> 
> > That has been my understanding too, merging this should make it public.
> 
> It's been over a month since I touched it, but I believe you now need to run `rake publish` to make it 'public' after committing. See the README and #74 for more information.

Ah ok.. Thanks Marty, will check it.

File: source/includes/_first-things-first.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ The API follows a typical RESTful pattern where it allows you to access collecti
 Some objects returned by the API may have sub-collections. For example, a tags collection for a given user object. These sub-collections are paginated in the same manner as top level collections.
 
 ```
-curl -X GET https://api.10000ft.com/api/v1/users \
+curl -X GET https://api.rm.smartsheet.com/api/v1/users \
```
Comment (KristenWebster) [inline source/includes/_first-things-first.md:29]: In other places you've changed `https` to `http`, does it need to be `http` here too?  Same for the other places in this PR

**Review by KristenWebster — COMMENTED** [review]
Have one question about `http` but overall looks good

File: source/includes/_first-things-first.md (hunk 1 above)
Comment (aiyer-smartsheet) [reply source/includes/_first-things-first.md:29]: OMG!!! 🙈  Good catch, the protocol should not change and should be set to 'https'. Let me update this.

**Review by KristenWebster — APPROVED** [review]

**Review by rchaudhry-smartsheet — APPROVED** [review]

**Pull request merged**